src/pages/docs/fournisseur-service/resource_server.md

```diff
@@ -0,0 +1,35 @@
+# Implémenter un Serveur de Ressources (SR)
+
+## Utilisation
+
+Un SR (ou Resource Server dans la terminologie OIDC) permet d'effectuer des actions au nom d'un utilisateur proConnecté et de renvoyer à un Fournisseur de Service des informations concernant l'utilisateur.
+
+## Fonctionnement
+
+1. L'utilisateur s'authentifie sur le FS via ProConnect selon le parcours classique. Dans le cadre de cette connexion, un `access_token` est renvoyé au FS.
+2. Le FS requête une route du SR pour effectuer une action. Par exemple, si le SR contient des informations de géolocalisation sur les utilisateurs, il peut s'agir de la route `https://resource-server.fr/api/users/locations`. Cet appel doit contenir l'`access_token` renvoyé par ProConnect lors de l'appel du FS au `token_endpoint`. L'usage est d'envoyer l'`access_token` en `Bearer` dans le header `Authorization`.
```

Comment:
issue: we should stop using non attributed domain for url example. Plz use example.com, example.org & example.edu for all url and email example.